### PR TITLE
Sumo logic Fetch Bug

### PR DESCRIPTION
**Pull request by Bargenish**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/28757

## Description
Fixed an issue with the ***fetch-incident*** command where not all events were pulled.

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/SumoLogic/Integrations/integration-SumoLogic.yml [hunk 1]
```diff
@@ -217,20 +217,23 @@ script:
...
-            var s = search(params.fetchQuery, lastRun.time, now, a2i(params.limit, defaultLimit), 0, params.timeZone, a2i(params.maxTimeout, 180), false,
+            var s = search(params.fetchQuery, lastRun.time, now, a2i(params.limit, defaultLimit), 0, params.timeZone, a2i(params.maxTimeout, 180), true,
                 a2i(params.sleepBetweenChecks, defaultSleep), params.fetchRecords);
             var incidents = [];
-
+            var currentFetch = lastRun.time;
             if (!params.fetchRecords && s.messages) {
                 for (var i=0; i<s.messages.length; i++) {
-                    var incident = {name: 'Incident from SumoLogic' + s.messages[i]._messageid,
+                    var incident = {name: 'Incident from SumoLogic' + s.messages[i]._receipttime,
```
Comment (yaakovi) [inline Packs/SumoLogic/Integrations/integration-SumoLogic.yml:226]: why did you change the incident name?

**Review by yaakovi — APPROVED** [review]

File: Packs/SumoLogic/Integrations/integration-SumoLogic.yml (hunk 1 above)
Comment (Bargenish) [reply Packs/SumoLogic/Integrations/integration-SumoLogic.yml:226]: a mistake, reverting this.
